detect_secrets/audit/report.py

```diff
@@ -0,0 +1,137 @@
+import hashlib
+import codecs
+import json
+from enum import Enum
+
+from .io import print_message
+from ..core.plugins.util import Plugin, get_mapping_from_secret_type_to_class
+from ..core.scan import _get_lines_from_file, _scan_line
+from ..core.potential_secret import PotentialSecret
+from ..plugins.base import BasePlugin
+
+
+class SecretClass(Enum):
+    TRUE_POSITIVE = 1
+    FALSE_POSITIVE = 2
+    UNKNOWN = 3
+
+    def from_boolean(is_secret: bool) -> Enum:
+        if is_secret == None:
+            return SecretClass.UNKNOWN
+        elif is_secret:
+            return SecretClass.TRUE_POSITIVE
+        else:
+            return SecretClass.FALSE_POSITIVE
+
+    def to_string(self) -> str:
+        return self.name
+
+    def get_prioritary(self, secret_class: str) -> Enum:
+        try:
+            to_compare = SecretClass[secret_class]
+        except:
+            return self
+        if to_compare.value < self.value:
+            return secret_class
+        else:
+            return self
```

Comment:
Let's use `detect_secrets.constants.VerifiedResult` instead of declaring another enum that pretty much does the same thing.

Reply:
Done!

Note that we include in the `report.py` file the methods `get_prioritary_verified_result` and `get_verified_result_from_boolean` to make comparations and conversions related to the `VerifiedResult` enum. This methods were specifically implemented to the reporting feature, but if you prefer to include them in the `VerifiedResult` object, I can do it.